examples/platforms/nrf528xx/src/radio.c

```diff
@@ -451,7 +461,8 @@ otError otPlatRadioReceive(otInstance *aInstance, uint8_t aChannel)
         nrf5FemEnable();
     }
 
-    nrf_802154_tx_power_set(sDefaultTxPower);
+    transmitPower = sDefaultTxPower < channelMaxPower ? sDefaultTxPower : channelMaxPower;
```

Comment:
Any need to check `sPowerInitialized` before using `sDefaultTxPower`?

Reply:
The behavior before this change is to use 0 as the implicit initialization and I prefer not to change it. As long as the max power table is passed, we have a guarantee to use the value in the power table.